vendor/acrios/acr-slc-100l-zhaga-dali.yaml

```diff
@@ -1,4 +1,4 @@
-name: LoRaWAN street light module
+name: LoRaWAN street light module (ZHAGA DALI)
```

Comment:
Why?

Reply:
This one we do need cause the device `acr-slc.yaml` (same vendor) is already called `LoRaWAN street light module`, so since this device uses the zhaga dali "protocol" it needs to be different.